CHANGELOG.md

```diff
@@ -62,6 +62,38 @@ prefix directory. For example `--prefix ~/.bcoin_whatever --network testnet`
 will create a directory structure of `~/.bcoin_whatever/testnet/` instead of
 `~/.bcoin_whatever`. Please update your directory structure accordingly.
 
+### Configuration Changes
+
+Wallet and Node are now separated and use different persistent configuration files.
+Some configuration options have moved to `wallet.conf`, which is stored in the prefix
+directory next to the existing `bcoin.conf`.
+
+All configuration options in `wallet.conf` will have the `wallet-`
+prefix added implicitly. The prefix is still needed when using CLI/ENV configuration
+methods. The wallet now has it's own HTTP server, so options such as
+`wallet-http-host` must also be specified along with `http-host`
+if you require this setting (required for Docker networking).
+
+Wallet-specific settings such as `api-key` and `wallet-auth`
+have moved to `wallet.conf`. If using CLI/ENV options, these are prefixed with `wallet-`.
```

Comment:
"these are prefixed" or "these should be prefixed"?

Reply:
No should, only do.